src/scheduler/gc_work.rs

```diff
@@ -826,6 +870,12 @@ pub trait ScanObjectsWork<VM: VMBinding>: GCWork<VM> + Sized {
                     // If an object supports edge-enqueuing, we enqueue its edges.
                     <VM as VMBinding>::VMScanning::scan_object(tls, object, &mut closure);
                     self.post_scan_object(object);
+
+                    #[cfg(feature = "count_live_bytes_in_gc")]
+                    closure
+                        .worker
+                        .shared
+                        .increase_live_bytes(VM::VMObjectModel::get_current_size(object));
```

Comment:
This should be hoisted before the `if` statement, because we will count the object's size regardless of whether it supports edge enqueuing or not.

Reply:
I have made this change.